pkg/cloud/packet/scope/machine.go

```diff
@@ -104,12 +103,10 @@ func (m *MachineScope) Close() error {
 	return m.patchHelper.Patch(context.TODO(), m.PacketMachine)
 }
 
-// Name returns the DOMachine name.
```

Comment:
Why remove this comment? It is a bit wrong, but should be there, or it will fail linting.

Reply:
Remove it because it mentioned `DOMachine` and as a comment is pretty useless

Reply:
Yeah, but golang linting requires a comment on all exported functions. Let's keep the comment there (corrected), even if it is useless.

Reply:
and we do not have the linter enabled and not all the functions has a comment

Reply:
You are right, but we will enable it eventually. Why remove something good?

Reply:
not sure it is good :D but it is back